back/apps/core-fca-low/src/controllers/oidc-client.controller.ts

```diff
@@ -158,15 +165,27 @@ export class OidcClientController {
       sp_name: spName,
     };
 
-    const { accessToken, idToken, acr, amr } = await this.oidcClient.getToken(
+    const { accessToken, idToken, claims } = await this.oidcClient.getToken(
       idpId,
       tokenParams,
       req,
       extraParams,
     );
 
+    let acr;
+    if (claims['acrs']) {
```

Comment:
issue: Since this behavior is specific to EntraID servers and is non-standard, a regular OIDC provider should not be able to send back these claims for ProConnect to interpret.
```suggestion
    if (idp.isEntraID && claims['acrs']) {
```